test/integration/default/controls/config_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+control 'splunkforwarder configuration' do
+  title 'should match desired lines'
+
+  describe file('/opt/splunkforwarder/etc/system/local/server.conf') do
+    its('type') { should eq :file }
+    it { should be_owned_by 'splunk' }
+    it { should be_grouped_into 'splunk' }
+    its('mode') { should cmp '0600' }
+  end
+
+  describe ini('/opt/splunkforwarder/etc/system/local/server.conf') do
+    its('httpServer.acceptFrom') { should eq '127.0.0.1' }
```

Comment:
Maybe we can add a check that is only listening in `127.0.0.1`

Reply:
I wrote a test but seems like the accepFrom acts as an ACL, not as a normal `listenOnly`.